package/MDAnalysis/core/universe.py

```diff
@@ -229,14 +278,14 @@ class Universe(object):
 
     Parameters
     ----------
-    topology : str, stream, `~MDAnalysis.core.topology.Topology`, `np.ndarray`, None
+    topology: str, stream, `~MDAnalysis.core.topology.Topology`, `np.ndarray`, None
```

Comment:
Builds ok locally. I actually removed the spaces before because it wasn't rendering nicely (locally) when I tried to autosummarise from the UserGuide.

<img width="675" alt="Screenshot 2019-09-23 at 3 33 36 AM" src="https://user-images.githubusercontent.com/31115101/65392083-08066780-ddb4-11e9-9201-d942a8adac8d.png">